dev-python/ipcalc/metadata.xml

```diff
@@ -1,14 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<longdescription lang="en">
```

Comment:
This is not *long* description. If short is enough, just drop it.

Reply:
Ok, I remove it. thanks